src/lib/plugin/utils.ts

```diff
@@ -69,3 +129,14 @@ export async function getUsedPlugins(layout: Layout.Layout): Promise<Plugin[]> {
     })
   }
 }
+
+// helpers
+
+function addErrorMessageContext(additionalMessage: string, error: Error): Error {
+  error.message = `${additionalMessage}\n\n${error.message}`
+  return error
+}
+
+function createGetManifestError(error: Error): Error {
+  return new Error(`An error occured whlie loading one of the plugins you are using.\n\n${error.message}`)
+}
```

Comment:
All this error stuff is pretty bad and ad-hoc. I would like to get to a better place:

- validation types/funcs from fp-ts
- error wrapping/context chains (Joyent has a decent option for that, old and still maintained, we may want to adopt it https://github.com/joyent/node-verror)